src/containers/VillagerContainer.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import VillagerList from '../components/VillagerList.jsx';
+import { getVillagers } from '../services/animalCrossingAPI.js';
+import { mungeVillagers } from '../services/animalCrossingAPI.js';
+
+
+export default class VillagerContainer extends Component{
+  state = {
+    isLoading: true,
+    villagers: []
+  };
+
+  componentDidMount = async () => {
+    const res = await getVillagers();
+    const villagers = mungeVillagers(res);
```

Comment:
nice. you can also just map through the response in the fetch and munge the data. great idea!